lib/controllers/feedback.js

```diff
@@ -12,7 +12,6 @@ module.exports = function(options) {
     }
 
     var parsedCreateIssueUrl = url.parse(options.issuesUrl, true);
-    parsedCreateIssueUrl.query.access_token = options.accessToken;
```

Comment:
I was just writing a comment that this hadn't been removed and now it has 😄 

Could we pass `options.issueUrl` straight to the `request()`?